### ISSUE TITLE
Add hard deadline to tasks

### ISSUE THREAD

**Issue by Kobzol** [mrlvsb/kelvin#567]
We should have the option to set a "hard deadline", where Kelvin won't allow students to submit anything after the deadline has passed.

**Comment by Dym03** [comment]
Is this still a wanted feature. Had 2 ideas. First would be another date input, for the hard deadline. Second would be just bool that would set if the deadline is hard or not. Second one would be easier, but could provide limited features for example last minute submits in tests, etc. Which could be solved by the first approach.

**Comment by Kobzol** [comment]
Yeah, I think that for some use-cases it makes sense. CC @geordi if you have any suggestions. I think that a bool would be enough.
